core-default/src/main/java/me/aurium/beetle/defaults/datacore/Simplification.java

```diff
@@ -0,0 +1,5 @@
+package me.aurium.beetle.defaults.datacore;
+
+public class Simplification {
```

Comment:
What does this do?

Reply:
Unimplemented but it exists basically for simplification of sql queries